src/main/java/uk/nhs/digital/nhsconnect/nhais/mapper/PersonNameMapper.java

```diff
@@ -0,0 +1,29 @@
+package uk.nhs.digital.nhsconnect.nhais.mapper;
+
+import org.hl7.fhir.r4.model.Parameters;
+import org.hl7.fhir.r4.model.Patient;
+import uk.nhs.digital.nhsconnect.nhais.model.edifact.PersonName;
+
+public class PersonNameMapper implements FromFhirToEdifactMapper<PersonName> {
+    private final static String PATIENT_PARAM = "patient";
+
+    public PersonName map(Parameters parameters) {
+
+        Patient patient = getPatient(parameters);
+
+        return PersonName.builder()
+                .surname(patient.getNameFirstRep().getFamily())
```

Comment:
Beware of these getXXXFirstRep() methods:

> getNameFirstRep
> public HumanName getNameFirstRep()
> Returns:
> The first repetition of repeating field name, **creating it if it does not already exist**

Is this really what we want?

Reply:
yes, that's fine. getNameFirstRep().getFamily() will throw NPE.
I'll add test for that + validation